docs/documentation/cl_options/index.rst

```diff
@@ -0,0 +1,40 @@
+Command Line Arguments
+======================
+
+What starting vimiv form the command line you have the ability to pass a number of
+different argument to vimiv. The general calling structure is depicted below:
+
+.. include:: synopsis.rstsrc
+
+Command Line Arguments
+----------------------
+
+The following is an exhaustive list of all available arguments:
+
+.. include:: options.rstsrc
+
+Examples
+--------
+
+In the following we present a few use cases of command line arguments.
+
+* Start in thumbnail view instead of image view
+    ``vimiv * --command "enter thumbnail" --command "enter library"``
```

Comment:
I would remove the `--command "enter library"` here as the text says "start in thumbnail view". You could add a second point stating "start in library view with the thumbnail grid displayed" for the combination if you like.

Reply:
I have improved the wording as you have suggested in e6a562414448933601ee2b9b8f0000c9fe266fc2.